code/controllers/subsystem/ticker.dm

```diff
@@ -56,13 +56,15 @@ var/datum/subsystem/ticker/ticker
 
 	var/news_report
 
+	var/list/prioritized_jobs = list()
```

Comment:
This isn't rocket science, define the var on the controller datum, you can then access it globally via SSJob.varname

please do this